config/locales/en.yml

```diff
@@ -6,7 +6,8 @@ en:
 
   service:
     name: Apply for qualified teacher status (QTS) in England
-    email: my-service@email
+    email: qts.enquiries@education.gov.uk
```

Comment:
Is this the right email? Looks good, just would like to double check in Slack if you haven't already.